api/src/main/java/com/google/sps/GameTimer.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GameTimer {
+
+  @PostMapping("/api/start")
+  public void timerInit(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
If this is to be implemented in the future, could you add some documentation indicating what the method will be used for

Reply:
do you mean like comments?

Reply:
do you mean comments?

Reply:
like [javadocs](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html) or something